### DISCUSSION TITLE
Prototyping desktop images with bootc

### DISCUSSION THREAD

**Discussion by castrojo** [bootc-dev/bootc#998] [General]
Hey ya'll!

I work on [Universal Blue](https://universal-blue.org/), which has been making desktop images via rpm-ostree built on Fedora. Over the past few years we've been building images on rpm-ostree. [Bazzite](https://bazzite.gg) is our most successful image. We basically `FROM fedora` and build a custom image for gaming handhelds. Here's [more videos](https://www.youtube.com/results?search_query=bazzite) but the basic idea is, if you can shove it in a containerfile, you can make what you want. 😄 

We've been transitioning to bootc and I wanted to share some of our experiences with the group. There's 3+ years of history here so let me start by showing some of our prototypes. Our latest one is built on CentOS Stream10:

https://www.youtube.com/watch?v=kFPqsK9lNPc

I've got a few years of backstory there but I'd also like to point out [HeliumOS](https://www.heliumos.org/) as well. Between both of our orgs we've got some decent experience so just thought I'd mention that if you're looking for help. 

There's so many things for us to discuss! I suspect this audience would be more interested in [ucore](https://github.com/ublue-os/ucore), which is our custom server image. bootc makes it pretty easy to just "make your own golden image" so we just made one. Looking forward to the discussions!

**Comment by imbev** [comment]
Hello, HeliumOS developer here!

I would like to echo the sentiment, bootc holds great potential for the Linux desktop. 

> bootc makes it pretty easy to just "make your own golden image"

This is the experience that I've had. bootc makes semi-declarative system configuration accessible to anyone familiar with OCI tooling.

Also open to asking and answering questions :)

**Comment by vrothberg** [comment]
> https://www.youtube.com/watch?v=kFPqsK9lNPc

I love it. Great work and thanks for sharing, @castrojo!

I am also toying with a bootc-based desktop in https://github.com/vrothberg/fedora-bootc-workstation.  I love the experience, especially the automated updates via Renovate bot (e.g., https://github.com/vrothberg/fedora-bootc-workstation/pull/33/files).
